src/report.test.js

```diff
@@ -60,7 +61,6 @@ describe('Report creation', () => {
       ];
 
       expect(_.isEqual(allowedMissingFields, diff)).toBe(true);
-      done();
```

Comment:
Since you've made this test async, I think you'll need some way of letting jest know that it needs this block needs to be run to pass. You can use `done` in combo with async or you could promisify the `r.send` function and await on it just for the test.
http://2ality.com/2017/05/util-promisify.html
We can make the send a promise and remove callback patterns in a separate PR later to even further increase the readability.